tests/transports-integrations/tests/integrations/test_litellm.py

```
@@ -358,6 +362,46 @@ def test_12_error_handling_invalid_roles(self, test_config):
         assert_valid_error_response(error, "tester")
         assert_error_propagation(error, "litellm")
 
+    def test_13_streaming(self, test_config):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_13_streaming`

Add return type annotation: `None`

(ANN201)